lighthouse-core/lib/network-request.js

```diff
@@ -205,6 +216,7 @@ module.exports = class NetworkRequest {
 
     this._updateResponseReceivedTimeIfNecessary();
     this._updateTransferSizeForLightrider();
+    this._updateFetchStatsForLightrider();
```

Comment:
can you also add a comment above https://github.com/GoogleChrome/lighthouse/blob/9716274b5d1a4c69a7d086fe4943ce6e3410a5c0/lighthouse-core/lib/network-request.js#L325

    // We're only at responseReceived at this point. This endTime will be redefined again.

and if you want to tweak the method comment to remind the reader we're only at `_onResponse`, i i think that'd be dope.